<launch>
    <arg name="jpg_topic" default="/rgb/image_raw/compressed"/>
    <arg name="depth_topic" default="/depth_to_rgb/image_raw"/>
    <arg name="calibration_path" default="$(find mapping_robot)/calibration.txt"/>
    <arg name="bag_path" default="./test.bag"/>
    <arg name="pose_path" default="./laser_poses.txt"/>
    <arg name="export_path" default="./test"/>
    
    <node pkg="mapping_robot" type="parser_node" name="parser_node" output="screen" required="true">
        <param name="jpg_topic"     type="string"   value="$(arg jpg_topic)" />
        <param name="depth_topic"     type="string"   value="$(arg depth_topic)" />
        <param name="calibration_path" type="string" value="$(arg calibration_path)"/>
        <param name="bag_path"  type="string"   value="$(arg bag_path)"/>
        <param name="pose_path"  type="string"   value="$(arg pose_path)"/>
        <param name="export_path"  type="string"   value="$(arg export_path)"/>
    </node>
</launch>
